tests/perf/README.md

```diff
@@ -7,7 +7,7 @@ ulimit -c 3000000
 # to increase max file descriptors
 ulimit -n 64000
 # to increase port list (sudo su) 
-echo 10000 61000 > /proc/sys/net/ipv4/ip_local_port_range
+echo 5000 65000 > /proc/sys/net/ipv4/ip_local_port_range
```

Comment:
It seems that the command above expands the range temporarily and the following is necessary to make the change persistent:
```
sudo sysctl -n net.ipv4.ip_local_port_range="5000 65000"
```
Please check this again and if so add to README

Reply:
ok to be permanent should be written also in /etc/sysctl.conf